start/sw/gui-app/electron.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Electron
+subsection: gui-app
+section: start-sw
+description: 
+---
+
+# Create an application using Electron
+
+[Electron][0] is cross platform framework for creating desktop applications using
+web technologies. It is developed by GitHub and released under the MIT license.
+
+## Installing the development packages
+
+Electron uses web technologies so before installing Electron you will need to
+install NodeJS and NPM. You can install them by running:
+
+```
+$ sudo dnf install nodejs
+```
+
+or by following the [Node.js](/tech/languages/nodejs/nodejs.html) tutorial.
+
+After installing NodeJS you can simply install electron using NPM or your preferred
+package manager for NodeJS. You can install it using NPM by running:
+
+```
+$ npm install --save-dev electron@latest
+```
+
+after installing nodejs and electron you can follow the [first application][1]
```

Comment:
Are there no Fedora specifics? Does the guide work fine with no glitches? I mean, for someone, who's going to do GUI app for the first time, is that a proper `quickstart`?
Is there some other / advanced one, for someone who's already experienced with other GUI toolkits? (I'm missing some references here.)